configure_reclient.py

```diff
@@ -146,7 +146,7 @@ def run_custom_py_post_configure(self):
     @staticmethod
     def download_linux_clang_toolchain():
         subprocess.check_call([
-            'python3',
+            sys.executable,
             f'{Paths.src_dir}/tools/clang/scripts/update.py',
             '--output-dir',
             f'{Paths.linux_clang_base_path}',
```

Comment:
use `os.path.join` to form paths?

Reply:
Hmm, since this works on Windows, even on Powershell, not sure what's the point. And if there is a need, then better to fix everywhere (this file has a lot of path references) -- and then I'd rather do it separately.